polaris-migrator/src/migrations/icons-v8-update-names/icon-renames.ts

```diff
@@ -835,6 +835,7 @@ export const iconRenames = {
   SidebarRightMinor: 'LayoutSidebarRightIcon',
   SidekickMajor: 'SidekickIcon',
   SidekickMinor: 'SidekickIcon',
+  SkeletonMajor: 'SkeletonIcon',
```

Comment:
Looks good! Our tests have every single icon in them lol. So if you want to add it to the test as well please do.

Reply:
Done! https://github.com/Shopify/polaris/pull/11318/commits/4dbcc903dea09f67ed05886b16875e2686ce2afd